kvui.py

```diff
@@ -35,6 +35,14 @@
 Config.set("kivy", "exit_on_escape", "0")
 Config.set("graphics", "multisamples", "0")  # multisamples crash old intel drivers
 
+from kivy.utils import platform
+if platform == 'linux':
+    #Linux has some weirdness with the touchpad by default... remove it
```

Comment:
I would prefer a comment link to the issue, so we can more easily find a path to judge when/whether the workaround code can be removed.